### ISSUE TITLE
Alt text for logo

### ISSUE THREAD

**Issue by AgaFra** [Problematy/goodmap#135]
**Is your feature request related to a problem? Please describe.**
People who are blind or visually impaired are unable to say what is in the place where the logo is located.

**Describe the solution you'd like**
Add alt text to the logo image.

**Comment by raven-wing** [comment]
Fixed with https://github.com/Problematy/platzky/pull/2
